app/models/concerns/ten_digit_goods_nomenclature.rb

```diff
@@ -30,7 +30,7 @@ module TenDigitGoodsNomenclature
 
     dataset_module do
       def by_code(code = '')
-        filter(goods_nomenclature_item_id: code.to_s.first(10))
+        filter(goods_nomenclatures__goods_nomenclature_item_id: code.to_s.first(10))
```

Comment:
I guess you're fully qualifying to enable this to be used with association/other kinds of join?

Reply:
We're now joining on tree nodes and theres a `goods_nomenclature_item_id` column on that table as wel so the old code results in ambiguity (and PG raises an exception)